controllers/errors.js

```diff
@@ -1,8 +1,25 @@
-exports.handle404s = (req, res, next) => {
-  res.status(404).send({ msg: "Cannot find what you wanted" });
+exports.handleCustomErrors = (req, res, next) => {
```

Comment:
This is great, it's just not handling custom errors, try and think about a more suitable name 👍